runners/direct-java/src/main/java/org/apache/beam/runners/direct/ViewEvaluatorFactory.java

```diff
@@ -77,7 +77,9 @@ public void processElement(WindowedValue<Iterable<InT>> element) {
       @Override
       public TransformResult finishBundle() {
         writer.add(elements);
-        return StepTransformResult.withoutHold(application).build();
+        return StepTransformResult.withoutHold(application)
+            .withAdditionalOutput(!elements.isEmpty())
```

Comment:
Given the change to a set of outputs that were produced, the method now takes an enum.